torchao/quantization/quant_api.py

```diff
@@ -389,7 +389,7 @@ def int4_weight_only(group_size=128, layout_type=TensorCoreTiledLayoutType(inner
          size is more fine grained, choices are [256, 128, 64, 32]
         `layout_type`: layout type for quantized tensor, default is `TensorCoreTiledLayoutType(inner_k_tiles=8)`
     """
-    def apply_int4_weight_only_quant(weight):
+    def apply_int4_weight_only_quant(weight, use_hqq=False):
```

Comment:
I just found that this flag is not used, so we don't really expose hqq to users right now, are you planning to create a new function for hqq? cc @mobicham

Reply:
My understand was that @HDCharles suggested putting it there and later turning it on by default. 
It is exposed though via `to_affine_quantized` https://github.com/pytorch/ao/pull/605/files#diff-a9708dc28f15bb9cf665417e6c66601f9e8e2f1f672d1858603b74fa879a3357R62
Let me know if there's another way of exposing it

Reply:
commented in https://github.com/pytorch/ao/pull/786#discussion_r1741153437